src/topp/IDFileConverter.cpp

```diff
@@ -329,6 +330,20 @@ class TOPPIDFileConverter :
       else if (in_type == FileTypes::IDXML)
       {
         IdXMLFile().load(in, protein_identifications, peptide_identifications);
+        // get spectrum_references from the mz data, if necessary:
+        if (!mz_file.empty())
```

Comment:
Only if mz_file is given, spectrum_reference annotation will be undertaken, analogue to how addMissingRTsToPeptideIDs is invoked if FileType is mzid. ok?